modules/react/icon/lib/SystemIcon.tsx

```diff
@@ -191,6 +191,7 @@ export const systemIconStencil = createStencil({
       ),
     },
   }),
+  modifiers: {},
```

Comment:
defaulting this to an empty object. If you extend a stencil and it's missing a property, typescript doesn't know to default to an empty object and crashes.